unit/testing-utils/require_symbol.cpp

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************\
+
+ Module: Unit test utilities
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+
+#include "require_symbol.h"
+#include "catch.hpp"
+
+const symbolt &require_symbol::require_symbol(
+  const symbol_tablet &symbol_table,
+  const irep_idt &symbol_identifier)
+{
+  const symbolt *found_symbol=symbol_table.lookup(symbol_identifier);
```

Comment:
How about `lookup_ref`? The dereferenced value is returned anyway.

Reply:
I dereference after `REQUIRE`ing it is not null. If this `REQUIRE` fails it throws an exception that fails the test. The point of this function is to fail the test if the symbol is not found.